libkbfs/tlf_journal.go

```diff
@@ -1461,13 +1457,12 @@ func (j *tlfJournal) getBlockData(id kbfsblock.ID) (
 type ErrDiskLimitTimeout struct {
 	timeout        time.Duration
 	requestedBytes int64
-	availableBytes int64
 	err            error
 }
 
 func (e ErrDiskLimitTimeout) Error() string {
-	return fmt.Sprintf("Disk limit timeout of %s reached; requested %d bytes, only %d bytes available: %+v",
-		e.timeout, e.requestedBytes, e.availableBytes, e.err)
+	return fmt.Sprintf("Disk limit timeout of %s reached; requested %d bytes: %+v",
```

Comment:
Ah, I put it back in. On an error, the available bytes may not be returned, but that can be accounted for.